force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -28,8 +28,12 @@ public with sharing class FlowControlLoopsCollections {
      * @param y The second number.
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
-    public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+    public static String helloWorld(Integer x, Integer y) { 
+        if  (x > y) {
+            return 'Hello World!';
+        }   else {
+            return null;
+        }               
```

Comment:
Consider doing a null check / checking for the invalid case before you do your comparison. It is convention to return early your invalid cases, errors, and to throw exceptions.